tests/tensor/test_type.py

```diff
@@ -326,3 +339,114 @@ def test_deprecated_kwargs():
         new_res = res.clone(broadcastable=(False, True))
 
     assert new_res.shape == (None, 1)
+
+
+def test_tensor_creator_helper():
+    res = tensor(shape=(5, None))
+    assert res.type == TensorType(config.floatX, shape=(5, None))
+    assert res.name is None
+
+    res = tensor(dtype="int64", shape=(5, None), name="custom")
+    assert res.type == TensorType(dtype="int64", shape=(5, None))
+    assert res.name == "custom"
+
+    # Test with positional arguments only
+    res = tensor(dtype="int64", shape=(5, None))
+    assert res.type == TensorType(dtype="int64", shape=(5, None))
+    assert res.name is None
```

Comment:
This is an exact duplicate of the block above

Reply:
You're right. I missed it from the changes between the first and second commits